Norm_end_to_end.Rmd

```diff
@@ -77,70 +75,106 @@ naomit <- assay(res, "raw_filled") |>
     na.omit()
 ```
 
-### PCA unsupervised 
+### PCA unsupervised
 
 Look for overall biases in batch, sample, injection index,...
 
-```{r unsupervised checks, include=FALSE}
-pca_res <- prcomp(t(log2(naomit)), scale = FALSE, center = TRUE)
+```{r unsupervised checks, include=TRUE, fig.width = 10, fig.height = 5}
+## note: scale (center and scale) on columns (features after the t() call) to
+## remove dependency on absolute abundances.
+vals <- assay(res, "raw_filled_imputed") |>
+    log2() |>
+    t() |>
+    scale(center = TRUE, scale = TRUE)
+
+pca_res <- prcomp(vals, scale = FALSE, center = FALSE)
 
 par(mfrow = c(1, 2), mar = c(4.5, 4.5, 4.5, 1))
-plot_pca(pca_res, pc_x = 1, pc_y = 2, col = col_source)
-legend("topleft", inset = c(0, -0.24), col = col_source,
-       legend = names(col_source), lty=1, lwd = 2, xpd = TRUE, ncol = 3,
+plot_pca(pca_res, pc_x = 1, pc_y = 2, pch = 21,
+         col = col_sample, bg = paste0(col_sample, 80),
+         labels = res$injection_index)
+legend("topleft", inset = c(0, -0.24), col = col_phenotype,
+       legend = names(col_phenotype), lty=1, lwd = 2, xpd = TRUE, ncol = 3,
        cex = 0.7, title = "Phenotype", title.cex = 0.8)
-plot_pca(pca_res, pc_x = 3, pc_y = 4, col = col_source)
+plot_pca(pca_res, pc_x = 3, pc_y = 4, pch = 21,
+         col = col_sample, bg = paste0(col_sample, 80),
+         labels = res$injection_index)
+
+## - clear separation of study (plasma) from QC Pool (serum) on PC1.
+## - separation based on phenotype on PC3.
+## - no relationship of PC1 to PC4 on injection index.
+
+## PCA without any imputed values to check PCA is not driven by imputation.
```

Comment:
would prefer the PCA on the imputed data (because it's supposedly more complete). I think we can drop the PCA below, since (IMHO) the results are similar.

Reply:
This will by quite dataset dependent though. (if we think about the NAFLD data), so I should maybe write a little sentence about that